version/version_test.go

```diff
@@ -0,0 +1,16 @@
+package version
+
+import "testing"
+
+func TestUserAGent(t *testing.T) {
+	rt = "go1.2.3"
+	goos = "linux"
+	goarch = "amd64"
+	version = "0.1"
+	googClient = "googleapi-go/0.1"
```

Comment:
Nit: this is not the line google api agent actually has, usually. 

Since google api libs are prone to changes, is it possible to do use the live import here + a regex match?

E.g. "google-api-go-client/[\.0-9]" or something?

Reply:
That way we'll be alerted if the agent changes.

Reply:
Yup, makes sense. Updated.